packages/app/src/components/ConceptPickerButton.vue

```diff
@@ -49,4 +60,18 @@ export default {
 </script>
 
 <style scoped>
+h4 {
+  font-family: Montserrat;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+.rect {
+  border-radius: 10px;
```

Comment:
Add this to the btn-concept-picker class

Reply:
Cannot do that as it is used in a tag.

Reply:
I see. Let's rename it to rounded-rect